### PR TITLE
K1J-467: Updated confirmation modal alert

### PR DESCRIPTION
**Pull request by mhornfeldt**

### CODE REVIEW

File: certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/certificatemodel/model/AlertType.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package se.inera.intyg.certificateservice.domain.certificatemodel.model;
+
+public enum AlertType {
+  INFO, OBSERVE, ERROR
```
Comment (mww-sopra-steria) [inline certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/certificatemodel/model/AlertType.java:4]: There is something similar already in the model that is called ElementMessageLevel. Could we do something in order to use the same concept? At least within the model?

**Review by mww-sopra-steria — CHANGES_REQUESTED** [review]
Looks good! Just one thought relating to AlertType to consider.

File: certificate-service/app/src/main/java/se/inera/intyg/certificateservice/application/certificate/dto/AlertDTO.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import lombok.Builder;
+import lombok.Value;
+import se.inera.intyg.certificateservice.application.certificate.dto.AlertDTO.AlertBuilder;
+import se.inera.intyg.certificateservice.domain.certificatemodel.model.MessageLevel;
+
+@JsonDeserialize(builder = AlertBuilder.class)
+@Value
+@Builder
+public class AlertDTO {
```
Comment (ChrilleSigerhed) [inline certificate-service/app/src/main/java/se/inera/intyg/certificateservice/application/certificate/dto/AlertDTO.java:13]: I think we need to update the AlertBuilder to AlertDTOBuilder?

**Review by ChrilleSigerhed — APPROVED** [review]
Minor DTO update needed, but everything else looks good. Well done

**Pull request merged**